flawfinder

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
Thanks for the comments!  However, the goal is to have this program run under both Python2 and Python3. Have you tested the changes to ensure they work on Python2?

Reply:
Hello @david-a-wheeler, thanks for the prompt reaction!

You're right, my PR will break Python 2 support and won't work for both, Python 2/3. That said, current code base "as-is" does **not** work for Python 3.6+. See my results below: 

<details><summary>Output when running flawfinder with Python 3.7.5</summary>

```bash
❯ ./flawfinder .
Flawfinder version 2.0.10, (C) 2001-2019 David A. Wheeler.
Number of rules (primarily dangerous function names) in C/C++ ruleset: 223
Examining ./test-patched.c
Examining ./buggy-latin1.c
Traceback (most recent call last):
  File "./flawfinder", line 2234, in <module>
    sys.exit(flawfind())
  File "./flawfinder", line 2226, in flawfind
    if process_files():
  File "./flawfinder", line 2063, in process_files
    process_file_args(files, patch_infos)
  File "./flawfinder", line 1797, in process_file_args
    maybe_process_file(f, patch_infos)
  File "./flawfinder", line 1742, in maybe_process_file
    maybe_process_file(os.path.join(f, dir_entry), patch_infos)
  File "./flawfinder", line 1766, in maybe_process_file
    process_c_file(f, patch_infos)
  File "./flawfinder", line 1508, in process_c_file
    text = "".join(my_input.readlines())
  File "/usr/local/Cellar/python/3.7.5/Frameworks/Python.framework/Versions/3.7/lib/python3.7/codecs.py", line 322, in decode
    (result, consumed) = self._buffer_decode(data, self.errors, final)
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xe9 in position 10: invalid continuation byte
```
</details>

This contribution was made based on the rationale that Python 2 is at EoL. Of course, if you intend to keep that level of support, my contribution can totally be discarded :).

Cheers,